backend/shared_configs/configs.py

```diff
@@ -83,9 +83,15 @@ def validate_cors_origin(origin: str) -> None:
         raise ValueError(f"Invalid CORS origin: '{origin}'")
 
 
-CORS_ALLOWED_ORIGIN = os.environ.get("CORS_ALLOWED_ORIGIN", "*").split(",") or ["*"]
+CORS_ALLOWED_ORIGIN = os.environ.get("CORS_ALLOWED_ORIGIN", "*")
+if CORS_ALLOWED_ORIGIN:
+    CORS_ALLOWED_ORIGIN = [
```

Comment:
It seems like the subsequent validation loop should just go in here since if we default to "*" that should never need validation.